docs/dcp2_system_design.rst

```diff
@@ -471,6 +471,45 @@ SHA-256 hashes of the data file's content.
    comparison
 
 
+External DRS File URIs
+~~~~~~~~~~~~~~~~~~~~~~
+
+Some files may be hosted by an external DRS repository and are not available for
+import to the repository. In these cases, the ``drs_uri`` property may be provided
+in the file descriptor to indicate this to the importer. A descriptor with
+this property set will cause a ``null`` value in the ``file_id`` column of the
+``…_file`` table row for the data file referenced in the descriptor, in any TDR
+snapshot containing the data file. [#]_,
+The importer will skip any attempt to import the externally referenced files.
+Downstream consumers of these file descriptors must be resilient to a ```NULL``
+TDR ``file_id`` column and fall back to using the DRS URI as provided in the
+descriptor ``content`` column.
+
+The ``drs_uri`` property must be a URI utilizing the ``drs://`` scheme. Content
+hashes are required for descriptors of this nature.
```

Comment:
The schema already requires `crc32c` and `sha256`. I don't think we should require the others. It would complicate the schema and those two are sufficient for any use case. One is fast and the other is virtually collision free. So I would say

```suggestion
The ``drs_uri`` property must be a URI utilizing the ``drs://`` scheme. As for regular files, descriptors for files at external DRS URIs are required to have the ``crc32c`` and ``sha256`` content
hashes.
```